hca/dss/__init__.py

```diff
@@ -206,14 +208,16 @@ def download(self,
         :param str version: The version to download, else if not specified, download the latest. The version is a
                             timestamp of bundle creation in RFC3339
         :param str download_dir: The directory into which to download
-        :param iterable metadata_files: One or more shell patterns against which all metadata files in the bundle will
+        :param iterable metadata_filter: One or more shell patterns against which all metadata files in the bundle will
                                         be matched case-sensitively. A file is considered a metadata file if the
                                         `indexed` property in the manifest is set. If and only if a metadata file
                                         matches any of the patterns in `metadata_files` will it be downloaded.
-        :param iterable data_files: One or more shell patterns against which all data files in the bundle will be
+        :param iterable data_filter: One or more shell patterns against which all data files in the bundle will be
                                     matched case-sensitively. A file is considered a data file if the `indexed` property
                                     in the manifest is not set. The file will be downloaded only if a data file matches
                                     any of the patterns in `data_files` will it be downloaded.
+        :param no_metadata: Whether or not to exclude metadata files. Cannot be set when --metadata-filter is also set.
```

Comment:
```suggestion
        :param no_metadata: Exclude metadata files. Cannot be set when --metadata-filter is also set.
```

I think that looks better in the `--help` output.